### DISCUSSION TITLE
OAuth Security

### DISCUSSION THREAD

**Discussion by baoskee** [plhery/node-twitter-api-v2#213] [Q&A]
Hey, what server do you use for `redirect_uri` in the OAuth spec. I combed through the codebase but could not find. This would be a concern whether my access tokens are saved by a 3rd party server.

**Comment by alkihis** [comment]
This lib does not encapsulate a server.
You should create a server by yourself in order to have your own `redirect_uri`.

An example can be found [here](https://github.com/alkihis/twitter-api-v2-user-oauth-flow-example).
